content/applications/inventory_and_mrp/maintenance/maintenance_setup.rst

```diff
@@ -0,0 +1,239 @@
+   any past or currently open maintenance requests, are available from the equipment category form.
+
+   Navigate to :menuselection:`Maintenance app --> Configuration --> Equipment Categories`, and
+   select a category to view. Locate the :guilabel:`Equipment` and :guilabel:`Maintenance` smart
+   buttons at the top of the form.
+
+   .. image:: maintenance_setup/maintenance-setup-smart-buttons.png
+      :align: center
+      :alt: Equipment and Maintenance smart buttons on equipment category form.
+
+   Click the :guilabel:`Equipment` smart button to view all equipment belonging to this category.
+   Click the :guilabel:`Maintenance` smart button to view any past or currently open maintenance
+   requests.
+
+Machines & tools
+----------------
+
+To add new equipment, navigate to :menuselection:`Maintenance app --> Equipment --> Machines &
+Tools`, and click :guilabel:`New`. This opens a blank equipment form.
+
+In the :guilabel:`Name` field, assign a name for the new equipment. In the :guilabel:`Equipment
+Category` field, click the drop-down menu and select which category this new equipment should belong
+to.
+
+If in a multi-company environment, click the drop-down menu in the :guilabel:`Company` field, and
+select the company in the database to whom the new equipment belongs.
+
+Next to the :guilabel:`Used By` option, select from one of three radio button options:
+:guilabel:`Department`, :guilabel:`Employee`, or :guilabel:`Other`.
+
+.. image:: maintenance_setup/maintenance-setup-new-equipment-left-side.png
+   :align: center
+   :alt: Left-hand side of information fields on new equipment form.
+
+If :guilabel:`Department` is selected, a :guilabel:`Department` field appears below the
+:guilabel:`Used By` option. Click the drop-down menu and select the department that uses this
+equipment.
+
+If :guilabel:`Employee` is selected, an :guilabel:`Employee` field appears below the :guilabel:`Used
+By` option. Click the drop-down menu and select the employee who uses this equipment.
+
+If the :guilabel:`Other` option is selected, both the :guilabel:`Department` and
+:guilabel:`Employee` fields appear below the :guilabel:`Used By` option. Click the drop-down menus
+for the respective fields, and choose which department and employee uses this equipment.
+
+In the :guilabel:`Maintenance Team` field, select the team responsible for this equipment. In the
+:guilabel:`Technician` field, select the team member/user responsible for this equipment.
+
+.. image:: maintenance_setup/maintenance-setup-new-equipment-right-side.png
+   :align: center
+   :alt: Right-hand side of information fields on new equipment form.
+
+In the :guilabel:`Used in location` field, enter the location wherein this equipment will be used,
+if not in an internal work center (e.g. in an office).
+
+In the :guilabel:`Work Center` field, click the drop-down menu, and select which work center this
+equipment will be used in.
+
+Product information
```

Comment:
do you think, to round out everything, that a brief mention/explanation about the 'Description' tab would come in handy here?